src/facade/async.ts

```diff
@@ -20,15 +20,15 @@ import { isFunction } from './lang';
  *  </div>`})
  * export class Zippy {
  *   visible: boolean = true;
- *   @Output() open: EventEmitter<any> = new EventEmitter();
- *   @Output() close: EventEmitter<any> = new EventEmitter();
+ *   @Output() open: EventEmitter<boolean> // used as interface
```

Comment:
pls replace this example with the one from docs
https://github.com/ngParty/ng-metadata/pull/66/files#diff-96cc378dcdef180641176501f42183c7R488